Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -209,12 +209,8 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _ChatChannelSwip
 }
 
 extension _ChatChannelListItemView {
-    var typingMemberString: String? {
-        guard let members = content.channel?.currentlyTypingMembers, !members.isEmpty else { return nil }
-        let names = members.compactMap(\.name).sorted()
-        return names.joined(separator: ", ") + " \(names.count == 1 ? "is" : "are") typing..."
-    }
-    
+
```

Comment:
the `getFormattedDate` helper in `updateContent` needs to be improved. The formatter object should be cached + the time should be formatted according to the current locale.

Reply:
✅ Done. It is cached now and it is possible to change the format by the customer 👍

Reply:
@nuno-vieira the format shouldn't be hardcoded like this. The time format depends on the device locale. We should use some of the existing time-format presets because they respect the locale of the device.